mifosng-android/src/main/java/com/mifos/mifosxdroid/adapters/SyncPayloadsAdapter.java

```diff
@@ -59,9 +59,6 @@ public void onBindViewHolder(RecyclerView.ViewHolder holder, int position) {
             ((ViewHolder) holder).tv_activation_date.setText(clientPayload.getActivationDate());
 
             switch (clientPayload.getGenderId()) {
-                case 22 :
-                    ((ViewHolder) holder).tv_gender.setText("Male");
```

Comment:
Why we are deleting this line of code?

Reply:
To remove redundancy. @jawidMuhammadi The piece of code, in case 22, is the same as in default. There is no need for extra cases as it will be implemented in the default case if there is no particular case.

Reply:
The functionality will remain the same even if we delete the above case. That's why we are deleting the above line of code.

Reply:
Actually, here, even the `switch case` is not a good place to use instead you can use simply `if-else` statement. Although it is always recommended to handle all cases ( `else` ) in a recyclerView adapter; because whenever we scroll, recycleView recycles the used Items of the screen for new items sometimes it can provide unwanted results or behavior if we don't handle all cases due to catch in memory.